### PR TITLE
Add Logstash Applications

### PR DESCRIPTION
**Pull request by gmfrasca**
## This Pull Request implements
Add Logstash ArgoCD Application manifests

## If migrating an Application to ArgoCD
- [x] I have completed the list of action items on [this page](https://aicoe.github.io/aicoe-cd/get_argocd_to_manage_your_app/)

### CODE REVIEW

File: manifests/overlays/prod/applications/data_hub/dh-stage-logstash.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  destination:
+    namespace: dh-stage-ingest
+    server: https://paas.stage.psi.redhat.com:443
+  project: data-hub
+  source:
+    path: logstash/overlays/stage
+    repoURL: https://github.com/AICoE/internal-data-hub.git
+    targetRevision: master
+  syncPolicy:
+    automated:
```
Comment (HumairAK) [inline manifests/overlays/prod/applications/data_hub/dh-stage-logstash.yaml:15]: Completely Optional: 
I'd suggest keeping auto-sync off _for now_ while you verify everything works as intended, then send a pr turning it on.

**Review by HumairAK — APPROVED** [review]

File: manifests/overlays/prod/applications/data_hub/dh-stage-logstash.yaml (hunk 1 above)
Comment (anishasthana) [reply manifests/overlays/prod/applications/data_hub/dh-stage-logstash.yaml:15]: This is on stage, so I think autosync is fine there. For prod we should definitely leave auto sync off for now.

**Review by anishasthana — APPROVED** [review]
lgtm

**Pull request merged**